interface/src/main.cpp

```diff
@@ -153,17 +153,17 @@ int main(int argc, const char* argv[]) {
     );
     QCommandLineOption displaysOption(
         "display",
-        "Preferred display.",
+        "Preferred display plugin. Valid options include \"Desktop\", \"OpenXR\", and \"OpenVR (Vive)\"",
         "displays"
     );
     QCommandLineOption disableDisplaysOption(
         "disableDisplayPlugins",
-        "Displays to disable. Valid options include \"OpenVR (Vive)\" and \"Oculus Rift\"",
+        "Display plugins to disable, separated by commas. Valid options include \"OpenXR\" and \"OpenVR (Vive)\"",
```

Comment:
All the other options seem to end with a period sign.
```suggestion
        "Display plugins to disable, separated by commas. Valid options include \"OpenXR\" and \"OpenVR (Vive)\".",
```